cmd/crowdsec-cli/utils.go

```diff
@@ -223,7 +223,11 @@ func UpgradeConfig(itemType string, name string, force bool) {
 		found = true
 		if v.UpToDate {
 			log.Infof("%s : up-to-date", v.Name)
+
 			if !force {
```

Comment:
if force is enabled, it won't download the data files again ? not sure it's what we want

Reply:
it will be download elsewhere anyway. It's to avoid download it twice.